include/scenefx/fx_renderer/fx_renderer.h

```diff
@@ -2,6 +2,7 @@
 #define SCENEFX_FX_OPENGL_H
 
 #include <wlr/backend.h>
+#include <wlr/render/allocator.h>
```

Comment:
Nit: if it isn't needed in the header we should include this in fx_renderer.c

Reply:
Done